cs3/app/provider/v1beta1/resources.proto

```diff
@@ -62,3 +65,12 @@ enum ViewMode {
   // then this mode MUST fall back to READ_WRITE.
   VIEW_MODE_PREVIEW = 4;
 }
+
+// Defines the valid targets for an app URL.
+enum Target {
+  TARGET_INVALID = 0;
+  // The app URL is to be opened on an iframe
+  TARGET_IFRAME = 1;
+  // The app URL is to be opened by redirecting to a new page
```

Comment:
`redirecting to a new page` sounds a little bit like a `HTTP 302`. 

When looking at existing descriptions of the "blank" target (eg. https://www.w3schools.com/tags/att_a_target.asp), `Opens the app URL in a new window or tab` could be a better fit.

Reply:
yep, the comment can be improved, `blank` better represents the intended behavior